website/docusaurus.config.js

```diff
@@ -48,8 +48,8 @@ const config = {
   headTags: [
     // Favicons declarations
     { tagName: 'link', attributes: { rel: 'apple-touch-icon', sizes: '180x180', href: '/img/apple-touch-icon.png' } },
```

Comment:
If this ends up working, I think these other links might need to be made relative too. I remember there being some nuance to this though, so hopefully this is the right change...

Reply:
Let's see how it goes 👍